data/transaction/construction_test.go

```diff
@@ -0,0 +1,178 @@
+package transaction
+
+import (
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/pubkeyConverter"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519"
+	"github.com/ElrondNetwork/elrond-go/crypto/signing/ed25519/singlesig"
+	"github.com/ElrondNetwork/elrond-go/hashing/blake2b"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+	"github.com/stretchr/testify/require"
+)
+
+const aliceSeedHex = "413f42575f7f26fad3317a778771212fdb80245850981e48b58a4f25e344e8f9"
```

Comment:
this is actually the private key (as how we are using now)

Reply:
Renamed.